BraveShared/BraveStrings.swift

```diff
@@ -1418,6 +1418,210 @@ extension Strings {
+        
+        public static let shortcutSettingsOpenNewTabTitle =
+            NSLocalizedString("shortcuts.shortcutSettingsOpenNewTabTitle",
+                              bundle: .braveShared,
+                              value: "Open New Tab",
+                              comment: "")
+        
+        public static let shortcutSettingsOpenNewTabDescription =
+            NSLocalizedString("shortcuts.shortcutSettingsOpenNewTabDescription",
+                              bundle: .braveShared,
+                              value: "Use Shortcuts to open a new tab via Siri - Voice Assistant",
+                              comment: "")
+        
+        public static let shortcutSettingsOpenNewPrivateTabTitle =
+            NSLocalizedString("shortcuts.shortcutSettingsOpenNewPrivateTabTitle",
+                              bundle: .braveShared,
+                              value: "Open New Private Tab",
+                              comment: "")
+        
+        public static let shortcutSettingsOpenNewPrivateTabDescription =
+            NSLocalizedString("shortcuts.shortcutSettingsOpenNewPrivateTabDescription",
+                              bundle: .braveShared,
+                              value: "Use Shortcuts to open a new private tab via Siri - Voice Assistant",
+                              comment: "")
+        
+        public static let shortcutSettingsClearBrowserHistoryTitle =
+            NSLocalizedString("shortcuts.shortcutSettingsClearBrowserHistoryTitle",
+                              bundle: .braveShared,
+                              value: "Clear Browser History",
+                              comment: "")
+        
+        public static let shortcutSettingsClearBrowserHistoryDescription =
+            NSLocalizedString("shortcuts.shortcutSettingsClearBrowserHistoryDescription",
+                              bundle: .braveShared,
+                              value: "Use Shortcuts to open a new tab via Siri - Voice Assistant",
+                              comment: "")
+        
+        public static let shortcutSettingsEnableVPNTitle =
+            NSLocalizedString("shortcuts.shortcutSettingsEnableVPNTitle",
+                              bundle: .braveShared,
+                              value: "Enable VPN",
+                              comment: "")
+        
+        public static let shortcutSettingsEnableVPNDescription =
+            NSLocalizedString("shortcuts.shortcutSettingsEnableVPNDescription",
+                              bundle: .braveShared,
+                              value: "Use Shortcuts to enable Brave VPN via Siri - Voice Assistant",
+                              comment: "")
+
+        public static let shortcutSettingsOpenBraveTodayTitle =
+            NSLocalizedString("shortcuts.shortcutSettingsOpenBraveTodayTitle",
+                              bundle: .braveShared,
+                              value: "Open Brave Today",
```

Comment:
Rename to `Brave News`

Reply:
Fix at https://github.com/brave/brave-ios/pull/3579/commits/6094980027fe2d3ea410dd134ff093f8876e7a3c